content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,151 @@
+  publish: true
+  round_robin: false
+  runtime_assets:
+  - cpu-checks-plugins
+  - sensu-ruby-runtime
+  secrets: null
+  stdin: false
+  subdue: null
+  subscriptions:
+  - system
+  timeout: 0
+  ttl: 0
+{{< /code >}}
+
+{{< code shell "JSON" >}}
+sensuctl check info check-cpu --format json
+{
+  "command": "check-cpu.rb -w 75 -c 90",
+  "handlers": [
+    "slack"
+  ],
+  "high_flap_threshold": 0,
+  "interval": 60,
+  "low_flap_threshold": 0,
+  "publish": true,
+  "runtime_assets": [
+    "cpu-checks-plugins",
+    "sensu-ruby-runtime"
+  ],
+  "subscriptions": [
+    "system"
+  ],
+  "proxy_entity_name": "",
+  "check_hooks": null,
+  "stdin": false,
+  "subdue": null,
+  "ttl": 0,
+  "timeout": 0,
+  "round_robin": false,
+  "output_metric_format": "",
+  "output_metric_handlers": null,
+  "env_vars": null,
+  "metadata": {
+    "name": "check-cpu",
+    "namespace": "default",
+    "created_by": "admin"
+  },
+  "secrets": null
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+Many of our [guides][6] demonstrate how to create, update, and retrieve resource definitions in this way.
+
+## Implement a monitoring as code workflow
+
+There's no one "correct" way to implement monitoring as code.
+To get started, try [SensuFlow][5], our git-based approach to managing Sensu resources.
+
+## Best practices for monitoring as code
+
+The repository of resource definitions can be any manner of source control repository (git, subversion, etc.). While a specific directory structure is not required, we will be suggesting one later in this document.
+
+- To maintain consistency, save all of your resources as only one file type: YAML or JSON.
+- Include all dependencies within a resource definition.
+For example, if a handler requires a dynamic runtime asset and a secret, include the asset and secret definitions with the definition for the handler itself.
+
+## Monitoring as code with SensuFlow
+
+[SensuFlow][5] is a git-based, prescriptive monitoring as code workflow that uses [sensuctl][2] (including [sensuctl prune][3]) to synchronize your monitoring and observability code with your Sensu deployments.
+
+{{% notice note %}}
+**NOTE**: SensuFlow is available for technical preview, and individual components in the workflow may change.
+Before you use SensuFlow in production, test it in a development environment or a dedicated test namespace in your current environment.
+{{% /notice %}}
+
+SensuFlow requires:
+
+- A monitoring code repository of Sensu resource definitions
+- A Sensu [role-based access control (RBAC)][4] service account with permission to manage all resources in your repository
+- A resource labeling convention to designate which resources the SensuFlow workflow should manage
+- Integration with your CI/CD system to run sensuctl commands as the service account user from the repository of resource definitions
+
+Use the [SensuFlow GitHub Action][1], a turnkey reference implementation, to create your own monitoring as code workflow.
+
```

Comment:
I did not add more in this section because I supposed the SensuFlow GitHub Actions instructions would live in the repo README, which makes sense to me.

But is there more to say here to help users be successful with trying out SensuFlow?

Reply:
[Slack comment from Caleb](https://sensu.slack.com/archives/C6AF34L6R/p1611871786134400)

Reply:
The published blog post: https://sensu.io/blog/monitoring-as-code-with-sensu-flow